node.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/node.h b/src/node.h
+index 543cf5b..f136158 100644
+--- a/src/node.h
++++ b/src/node.h
+@@ -404,7 +404,7 @@ struct node_module {
+   struct node_module* nm_link;
+ };
+ 
+-node_module* get_builtin_module(const char *name);
++NODE_EXTERN node_module* get_builtin_module(const char *name);
```

Comment:
Could this be applied in the node fork at https://github.com/electron/node instead of a local patch?